src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -64,7 +64,6 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord)
                 otlpLogRecord = new OtlpLogs.LogRecord
                 {
                     TimeUnixNano = (ulong)logRecord.Timestamp.ToUnixTimeNanoseconds(),
```

Comment:
lets just put a todo to store categoryName into attributes, to unblock this PR. OTLP LogExporter is non-stable anyway, so dont need to be blocked by this.

Reply:
+1

Reply:
Maybe `Attributes["dotnet.ilogger.categoryname"]`, based on the spirit of https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#appendix-a-example-mappings and https://github.com/open-telemetry/opentelemetry-specification/blob/33e51e72276c4aa1dbe4fba84b4b05944ac5c241/specification/resource/semantic_conventions/README.md#telemetry-sdk.

Reply:
or wait for resolution on https://github.com/open-telemetry/opentelemetry-specification/issues/2398